src/App.js

```diff
@@ -5,24 +5,15 @@ import HomePgae from './pages/HomePage';
 import QuotePage from './pages/QuotePage';
 import Navbar from './components/Navbar';
 
-class App extends React.Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-  }
-
-  render() {
-    return (
-      <>
-        <Navbar />
-        <Routes>
-          <Route path="/" element={<HomePgae />} />
-          <Route path="/calculator" element={<CalculatorPage />} />
-          <Route path="/quote" element={<QuotePage />} />
-        </Routes>
-      </>
-    );
-  }
-}
+const App = () => (
+  <>
+    <Navbar />
+    <Routes>
+      <Route path="/" element={<HomePgae />} />
+      <Route path="/calculator" element={<CalculatorPage />} />
+      <Route path="/quote" element={<QuotePage />} />
+    </Routes>
+  </>
+);
```

Comment:
### OPTIONAL
- [ ] It will be better if you fix the typo in HomePgae component.
- [ ] Also when ı run `npm start` page server starts with this 'http://localhost:3000/math-magicians' link. For ı see home page ı should click on the home link. It can be required in the next milestone when deploying the project. So please fix it.